frontend/src/components/configurate-episode/components/create-episode-form/components/record-preview-control/record-preview-control.tsx

```diff
@@ -64,12 +61,17 @@ const RecordPreviewControl: React.FC<Props> = ({
             onChange={handleChange}
           />
         </label>
-        <span className={styles.recordPreview}>{currentRecord || liveRecordName}</span>
+        <span className={styles.recordPreview}>{currentRecord ? currentRecord : fileUrl ? 'current episode' : ''}</span>
```

Comment:
```suggestion
        <span className={styles.recordPreview}>{currentRecord ?? fileUrl ? 'current episode' : ''}</span>
```

Reply:
Also maybe let's try 
currentRecord ?? (fileUrl && 'current episode')

Reply:
The shortest variant will be: The shortest variant will be: currentRecord || (fileUrl ? 'current episode' : '')

Reply:
> The shortest variant will be: currentRecord || (fileUrl ? 'current episode' : '')

I think `??` more suitable than `||`

Reply:
?? doesn't work for this case